docs/Información del servicio/Story board.mdx

```diff
@@ -0,0 +1,124 @@
+## Historial de versiones
+
+| Versión | Fecha      | Diferencia entre versiones                                                                                     |
+| ------- | ---------- | ---------------------------------------------------------------------------------------------------------------|
+| 1.0     | 2024-03-14 | Versión inicial del documento.                                                                                 |
+                                               |
+
+## StoryBoard CyC
+
+
+### 1.1 Inicio
+![Inicio](/img/storyBoard/Cell-Row-0-Col-0.png)
+
+Inicio referenciando a los simpsons como introducción.
+
+### 1.2 Frente a la sede de CyC
+![Frente a la sede de Cirio y Costal](/img/storyBoard/Cell-Row-0-Col-1.png)
+
+En frente de la sede de cirio y costal se encuentran varios personajes de los simpsons esperando la cola para poder recibir 
+alimentos y otros recursos.
+
+
+### 1.3 Homer pide su pedido
+![Homer pide su pedio](/img/storyBoard/Cell-Row-0-Col-2.png)
+
+Homer Simpson pide los alimentos que solicitó hace un par de días y nuestro Avatar AL se dispone a buscarlos.
+
+
+### 1.4 Error en Excel
+![Error en excel](/img/storyBoard/Cell-Row-0-Col-3.png)
+
+Al abre el excel y se encuentra en una situación comprometida ya que el excel dejo de funcionar.
+
+### 1.5 AL piensa en la solución
+![AL piensa en la solución](/img/storyBoard/Cell-Row-0-Col-4.png)
+
+AL recuerda que existe un grupo llamado Harmony que crea soluciones tecnológicas para diferentes ONG.
+
+
+### 1.6 La solución a nuestros problemas
+
+![La solucion a nuestros problemas](/img/storyBoard/Cell-Row-0-Col-5.png)
+
+Esta imagen muestra AL junto a un portátil mostrando una demo de nuestra página.
+
+### 1.7 Se muestra la aplicacion
+![Se muestra la aplicacion](/img/storyBoard/Cell-Row-0-Col-6.png)
+
+Se muestra la aplicación para mostrar la interfaz y el salto de calidad respecto al excel.
+
+
+### 1.8 Cierre del story board
+![Cierre de la aplicacion](/img/storyBoard/Cell-Row-0-Col-7.png)
+
+AL junto a homer celebran la aplicacion.
+
+### 1.9 Cierre del story board
+![Cierre de la aplicacion](/img/storyBoard/Cell-Row-0-Col-8.png)
+
+Cierre del storyboard
+
+
+
+
+
+## StoryBoard ACAT
+
+
+### 2.1 Inicio
+![Imagen 1](/img/storyBoard/Cell-Row-1-Col-0.png)
+
+Inicio referenciando a los simpsons como introducción.
+
+
+
+### 2.2 AL y el psicologo
+![Imagen 2](/img/storyBoard/Cell-Row-1-Col-1.png)
+
+AL pregunta al psicologo si sabe quien ha sido rehabilitado.
+
+### 2.3 Aparece el registro de ACAT
+![Imagen 3](/img/storyBoard/Cell-Row-1-Col-2.png)
+
+Aparece el registro de ACAT demostrando que es inficiente ya que no aparece.
+
+
+
+### 2.4 Al quiere mostrar nuestra aplicacion
+![Imagen 4](/img/storyBoard/Cell-Row-1-Col-3.png)
+
+AL quiere mostrar nuestra aplicacion.
+
+
+
+### 2.5 Aparece nuestra aplicacion
+![Imagen 5](/img/storyBoard/Cell-Row-1-Col-4.png)
```

Comment:
We have been told in previous presentations that we should avoid using low quality images. I recommend editing the ACAT window elsewhere to make it more visible. In case it still looks bad, you can add a vignette with even more zoom in which you only see what you want to show of ACAT instead of the whole page.